imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```diff
@@ -436,62 +443,87 @@ num_completed:
   FIELDNAM: Number of spins completed
   LABLAXIS: number of spins completed
 
-acq_end:
+acq_start_sec:
+  <<: *default_uint32
+  DEPEND_0: epoch
+  VAR_TYPE: support_data
+  CATDESC: Acquisition Start
+  FIELDNAM: Acquisition Start Seconds
+  UNITS: ns
```

Comment:
Is this actually ns, or is it (s)? The variable is named "seconds"

Reply:
I have ns on the page that Colin and I worked on together, but yeah the naming seems weird. I'll double check.

Reply:
Discussed this with Lo today. The acq start seconds is in units of seconds and the acq start sub seconds is in units of microseconds after a 1e-6 conversion from the raw to EU value. I was thinking about this more after our meeting and I will need to follow-up with them about where they want the EU conversion done because there is no L1B spin product.

Reply:
FYI that this other PR I was reviewing is what made me think of some of this too.
https://github.com/IMAP-Science-Operations-Center/imap_processing/pull/1346

There it looks like Tim is doing it in L1a right away. I would consider EU slightly different than a straight unit conversion with multiplying/dividing by a factor to get something from nano to micro or something like that 🤷 (EU is more like a calibration factor applied in my mind)

Reply:
@greglucas Yeah, sorry my comment about the EU doesn't make a ton of sense with what I said here. There are EU conversions involving voltages in the spin packet which I was also thinking about but didn't mention in my comment. So I need to figure out if/where the voltage EU conversions should be put into a dataset.

@frahmanif Thanks for checking on this!